addons/mail/tests/test_mail_channel.py

```diff
@@ -434,341 +408,3 @@ def test_multi_company_chat(self):
                 allowed_company_ids=self.company_admin.ids
             ).channel_get(self.partner_employee_c2.ids)
             self.assertTrue(initial_channel_info, 'should be able to chat with multi company user')
-
-    def test_multi_company_message_post_notifications(self):
```

Comment:
This test should probably be kept.

Reply:
I do not think, I wrote this test for one multi-company bug on the "email" mail channel 1b8c4d98d3d8dc62e4e9f017d92c8d14e3421b90 but it's not really useful anymore